### PR TITLE
setTimeout() instead of InteractionManager

### PR DESCRIPTION
**Pull request by gmlion**
In componentDidMount() of index.js, using a timeout resolves a problem
with scrollView.scrollTo() not working

### CODE REVIEW

**Comment by skv-headless** [comment]
Why did you change other lines? Here https://facebook.github.io/react-native/docs/timers.html is written that better to use setTimeout with timer mixin.

**Comment by gmlion** [comment]
I didn't, it's only automatic formatting...sorry for the noise

**Comment by skv-headless** [comment]
@gmlion could you please update PR?

File: index.js [hunk 1]
```diff
@@ -65,12 +65,13 @@ const ScrollableTabView = React.createClass({
   },
 
   componentDidMount() {
-    InteractionManager.runAfterInteractions(() => {
+    setTimeout(() => {
       if (this.scrollView && Platform.OS === 'android') {
+        console.log("Scrolling to " + this.props.initialPage + " with containerWidth " + this.state.containerWidth);
```
Comment (skv-headless) [inline index.js]: Please remove console.log

File: index.js [hunk 2]
```diff
@@ -65,12 +65,13 @@ const ScrollableTabView = React.createClass({
   },
 
   componentDidMount() {
-    InteractionManager.runAfterInteractions(() => {
+    setTimeout(() => {
```
Comment (skv-headless) [inline index.js]: Use this.setTimeout like in documentation

**Comment by skv-headless** [comment]
@gmlion I dont know what is working `setTimeout` or `IntreractionManager`. Can you please leave both to be sure.

**Comment by gmlion** [comment]
Upon further testing, I found InteractionManager alone working on development configuration, but I still need setTimeout for the release configuration. I'm doing a commit with both anyway.

Also, I have found the setTimeout initially on stackoverflow, but I see it's mentioned also here (albeit for a slightly different problem I think) https://github.com/skv-headless/react-native-scrollable-tab-view/issues/366, so the credits for the solution are @MirGhojam

**Comment by skv-headless** [comment]
@gmlion hope it will work for everyone.

**Pull request merged**